src/riot-rs-runqueue/src/runqueue.rs

```diff
@@ -70,7 +71,12 @@ impl<const N_QUEUES: usize, const N_THREADS: usize> RunQueue<{ N_QUEUES }, { N_T
     }
 
     fn ffs(val: usize) -> u32 {
-        USIZE_BITS as u32 - val.leading_zeros()
+        #[allow(
+            clippy::cast_possible_truncation,
+            reason = "all targeted platforms have 32-bit wide pointers at most"
+        )]
+        let ffs = USIZE_BITS as u32 - val.leading_zeros();
```

Comment:
"All targeted platforms' bit widths fit in a u32" (they'd even fit in an u8, because 32 < 255)

Also, why not `const { USIZE_BITS.try_into().expect("All targeted platforms' bit widths fit in a u32) }`? (OK, that doesn't work *yet* without some nightly-only features ... we'd need it to be const so that clippy doesn't complain about a run-time exception).